src/lang/en-US.ts

```diff
@@ -1092,6 +1092,13 @@ const Workflows: ResolvedIntlConfig['messages'] = {
 
     'workflows.guards.admin.title': `Missing required ${CommonMessages['terms.permissions']}`,
     'workflows.guards.admin.message': `You must have the admin capability to at least one prefix to create a {entityType}. Please contact an administrator to request access.`,
+
+    'workflows.advancedSettings.title': `Advanced Options`,
+    'workflows.disable.autoEnabledAlert.title': `We have auto enabled this {entityType}`,
+    'workflows.disable.autoEnabledAlert.message': `To keep this {entityType} disabled you will need to disable before ${CTAs['cta.saveEntity']}`,
+    'workflows.disable.title': `Enable {entityType}`,
+    'workflows.disable.message': `Use to toggle the processing of your {entityType}. Setting takes effect only after publishing.`,
```

Comment:
_Use to_ seems superfluous; the verb _toggle_ could open the sentence. However, this content is confusingly vague as it is unclear what _toggling the processing of an task_ effectively means.

Reply:
I'll make toggle the first word. It is vague on purpose mainly because we don't really ever say exactly what disable/enable is controlling so that is why I left it was just an open ended "toggle this thing" kinda message.

Reply:
Understood. The content is a little _too_ unclear IMO. In the interest of brainstorming, here are the options that come to mind:

- Control whether your entity processes data.

- Control whether your entity is processing data.

- Control the processing of your entity's data.

- Control whether your entity is active.

- Control whether your entity is actively processing data.

- Control whether your entity is disabled.

This is the result of a quick stream of consciousness. It may be worthwhile to tag Dave and/or Phil to see if they have any suggestions.

Reply:
Also, I would recommend starting the second sentence with the word _this_: _This setting takes effect only after publishing._ Variations of this sentence are listed below.

- This setting takes effect when your entity is published.

- This setting will take effect when your entity is published.

- This setting takes effect when your changes are published.

- This setting will take effect when your changes are published.